tests/klay_test_blockchain_test.go

```diff
@@ -166,6 +166,9 @@ func (bcdata *BCData) prepareHeader() (*types.Header, error) {
 		Governance: common.Hex2Bytes("b8dc7b22676f7665726e696e676e6f6465223a22307865373333636234643237396461363936663330643437306638633034646563623534666362306432222c22676f7665726e616e63656d6f6465223a2273696e676c65222c22726577617264223a7b226d696e74696e67616d6f756e74223a393630303030303030303030303030303030302c22726174696f223a2233342f33332f3333227d2c22626674223a7b2265706f6368223a33303030302c22706f6c696379223a302c22737562223a32317d2c22756e69745072696365223a32353030303030303030307d"),
 		Vote:       common.Hex2Bytes("e194e733cb4d279da696f30d470f8c04decb54fcb0d28565706f6368853330303030"),
 	}
+	if bcdata.bc.Config().IsKIP71ForkEnabled(num) {
```

Comment:
Is this fine to check fork enabled with parent block number here?

Reply:
Ahh, child block number should be checked here. Thanks!